generators/client/templates/angular/src/main/webapp/app/admin/admin.route.ts.ejs

```diff
@@ -52,17 +50,28 @@ const ADMIN_ROUTES = [
     <%_ if (websocket === 'spring-websocket') { _%>
     trackerRoute,
     <%_ } _%>
-    <%_ if (!skipUserManagement) { _%>
-    ...userMgmtRoute,
-    <%_ } _%>
     metricsRoute
 ];
 
-export const adminState: Routes = [{
+export const adminState: Routes = [
+<%_ if (!skipUserManagement) { _%>
+  {
+    path: 'user-management',
+    data: {
+      authorities: ['ROLE_ADMIN']
+    },
+    canActivate: [UserRouteAccessService],
+    loadChildren: () => import('./user-management/user-management.module').then(m => m.UserManagementModule)
+  },
+<%_ } _%>
+  {
     path: '',
     data: {
-        authorities: ['ROLE_ADMIN']
+      authorities: ['ROLE_ADMIN']
     },
     canActivate: [UserRouteAccessService],
     children: ADMIN_ROUTES
-}];
+  }
+];
+
+export const AdminRouting: ModuleWithProviders = RouterModule.forChild(adminState);
```

Comment:
Current style in generated app is to use `RouterModule.forChild` directly in modules `import` array. I think we should keep this unified style. Maybe this change is caused by my incorrect `RouterModule` remove comment. Sorry again about that.

Reply:
References I looked used that way, I really don't what ModuleWithProviders do.
But worked without it.
Thanks.

Reply:
I meant here and in `user-management.route.ts` comment that currently `RouterModule.forChild` is executed directly in modules `imports` array. So my suggestion was to:
* revert defining `AdminRouting` variable and use `RouterModule.forChild` in `admin.module.ts` as it was before
* remove variable `UserManagementRouting` and use `RouterModule.forChild` directly in `user-management.module.ts` in this way:
```
    imports: [
        <%= angularXAppName %>SharedModule,
        RouterModule.forChild(userManagementRoute)
    ],
```

Reply:
Sorry about that.

Take a look at:
https://github.com/jhipster/generator-jhipster/pull/10408#discussion_r324798539

Already been done at:
https://github.com/jhipster/generator-jhipster/blob/0c2a28ba9d41b36412b4de244aebe1f015144ae1/generators/client/templates/angular/src/main/webapp/app/app-routing.module.ts.ejs

What approach should I take?